tests/unit/amo/pages/TestCollectionList.js

```diff
@@ -83,9 +90,15 @@ describe(__filename, () => {
 
     fakeDispatch.resetHistory();
 
-    renderComponent({ store });
+    renderComponent({ store, errorHandler });
 
-    sinon.assert.notCalled(fakeDispatch);
+    sinon.assert.neverCalledWith(
```

Comment:
You could actually do the same thing here as you did above with `sinon.assert.neverCalledWithMatch(fakeDispatch, { type: FETCH_USER_COLLECTIONS })`, and then you don't need to pass `errorHandler` into the render method at all.

Reply:
I like this, because it's more declarative. It seems that we should either memoize the action itself, rather than rebuild it OR simply assert the type, since we don't actually care about the other values of the payload.